src/spacel/agent/mixin/cloudwatch_logs.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from spacel.agent.mixin.base import BaseMixinWriter
+
+logger = logging.getLogger('spacel')
+
+
+class SetupCloudWatchLogs(BaseMixinWriter):
+    def __init__(self, systemd, meta):
+        super(SetupCloudWatchLogs, self).__init__(systemd)
+        self._meta = meta
+
+    def logs(self, manifest):
+        self._docker_logs(manifest)
+        # TODO: self._journald_logs ?
```

Comment:
Is this feature still worth it?

The annoying bit is getting https://github.com/saymedia/journald-cloudwatch-logs baked into the AMI: go binary without a PPA.